tests/utils/axis-utils-tests.js

```diff
@@ -32,7 +32,7 @@ test('axis-utils #getTicksTotalFromSize', t => {
 
 test('axis-utils #getTickValues', t => {
   const scale = scaleLinear().domain([0, 1]).range(['red', 'blue']);
-  t.deepEqual(getTickValues(scale, 10, false),
+  t.deepEqual(getTickValues(scale, 10, false).map(d => Math.round(d * 1000) / 1000),
```

Comment:
I was with @apercu when he changed the values to [0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1] from [0, 0.1, 0.2, 0.30000000000000004, 0.4, 0.5, 0.6000000000000001, 0.7000000000000001, 0.8, 0.9, 1], because that's what the operation in the test returned then. Lo and behold, a few days later, the same operation yields [0, 0.1, 0.2, 0.30000000000000004, 0.4, 0.5, 0.6000000000000001, 0.7000000000000001, 0.8, 0.9, 1].

What will it be tomorrow? your guess is as good as mine because not only is this a highly unpredictable float division, but one which depends on how d3 implements it. 

for those reasons, I think that comparing the result to either [0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1] or [0, 0.1, 0.2, 0.30000000000000004, 0.4, 0.5, 0.6000000000000001, 0.7000000000000001, 0.8, 0.9, 1] is going to fail eventually. 
comparing to [0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1] with a tolerance of ±0.001 which is what this codes does ensures that the behavior we want is right, independently of the exact output of the float divisions.